python/ribasim/ribasim/model.py

```diff
@@ -738,3 +741,67 @@ def _add_allocation(self, uds):
             uds[varname] = da
 
         return uds
+
+    def to_fews(
+        self,
+        region_home: str | PathLike[str],
+        network: bool = True,
+        results: bool = True,
+    ) -> None:
+        """
+        Write the model network and results into files used by Delft-FEWS.
+
+        ** Warning: This method is experimental and is likely to change. **
+
+        To run this method, the model needs to be written to disk, and have results.
+        The Node and Link tables are written to shapefiles in the Config directory.
+        The results are written to NetCDF files in the Modules directory.
+        The netCDF files are NetCDF4 with CF-conventions.
+
+        Parameters
+        ----------
+        region_home: str | PathLike[str]
+            Path to the Delft-FEWS REGION_HOME directory.
+        """
+        region_home = DirectoryPath(region_home)
+        if network:
+            self._network_to_fews(region_home)
+        if results:
+            self._results_to_fews(region_home)
+
+    def _network_to_fews(self, region_home: DirectoryPath) -> None:
+        """Write the Node and Link tables to shapefiles for use in Delft-FEWS."""
+        df_link = self.link.df
+        df_node = self.node_table().df
+        assert df_link is not None
+        assert df_node is not None
+
+        network_dir = region_home / "Config/MapLayerFiles/{ModelId}"
+        link_path = network_dir / "{ModelId}Links.shp"
```

Comment:
Shapefiles truncate our column names:

> Normalized/laundered field name: 'from_node_id' to 'from_node_'

Reply:
Indeed. The FEWS Config already uses the truncated column names. FEWS also accepts GeoJSON but that is probably not ideal either given the lack of CRS support (assuming they use the latest standard). Hopefully they'll support GeoPackage or another format without these issues. Obligatory http://switchfromshapefile.org/.

Pyogrio does warn about this truncation by the way. We should check if we can silence that for this call, since we know.